README.md

```diff
@@ -42,6 +42,7 @@ Having a bug tracking system doesn't make bug handling efficient and CI/CD doesn
 - How quickly can you setup a new local test environment for the product? (minutes / hours / days)
 - How quickly can you respond to security issues in the code or dependencies?
 - Are all developers allowed to have local admin access of their computers?
+- Do you have any sort of technical principles or vision?
```

Comment:
I wonder if there's a more open way to phrase this. I guess most companies would at least think they have some technical principles. Got any ideas?

Reply:
Good point. I think this could be phrased as "Tell me about your technical principles or vision."